de-lib/src/main/java/org/iplantc/de/apps/client/AppDetailsView.java

```diff
@@ -61,10 +67,13 @@
 
         String detailsLabel();
 
+        @JsIgnore
         SafeHtml getAppDocError(Throwable caught);
```

Comment:
Only primitives (except long), String, Double, Boolean, and classes/interfaces annotated with `@JsType` are exported by GWT for use in JS. Without this `@JsIgnore` annotation, the GWT compiler will print a warning that this method will generate extra code to create a JS entrypoint, but it won't be usable in native JS.